A1:
  x: -0.175
  y: 0.175
  z: 0.0
A2:
  x: -0.125
  y: 0.175
  z: 0.0
A3:
  x: -0.075
  y: 0.175
  z: 0.0
A4:
  x: -0.025
  y: 0.175
  z: 0.0
A5:
  x: 0.025
  y: 0.175
  z: 0.0
A6:
  x: 0.075
  y: 0.175
  z: 0.0
A7:
  x: 0.125
  y: 0.175
  z: 0.0
A8:
  x: 0.175
  y: 0.175
  z: 0.0

B1:
  x: -0.175
  y: 0.125
  z: 0.0
B2:
  x: -0.125
  y: 0.125
  z: 0.0
B3:
  x: -0.075
  y: 0.125
  z: 0.0
B4:
  x: -0.025
  y: 0.125
  z: 0.0
B5:
  x: 0.025
  y: 0.125
  z: 0.0
B6:
  x: 0.075
  y: 0.125
  z: 0.0
B7:
  x: 0.125
  y: 0.125
  z: 0.0
B8:
  x: 0.175
  y: 0.125
  z: 0.0

C1:
  x: -0.175
  y: 0.075
  z: 0.0
C2:
  x: -0.125
  y: 0.075
  z: 0.0
C3:
  x: -0.075
  y: 0.075
  z: 0.0
C4:
  x: -0.025
  y: 0.075
  z: 0.0
C5:
  x: 0.025
  y: 0.075
  z: 0.0
C6:
  x: 0.075
  y: 0.075
  z: 0.0
C7:
  x: 0.125
  y: 0.075
  z: 0.0
C8:
  x: 0.175
  y: 0.075
  z: 0.0

D1:
  x: -0.175
  y: 0.025
  z: 0.0
D2:
  x: -0.125
  y: 0.025
  z: 0.0
D3:
  x: -0.075
  y: 0.025
  z: 0.0
D4:
  x: -0.025
  y: 0.025
  z: 0.0
D5:
  x: 0.025
  y: 0.025
  z: 0.0
D6:
  x: 0.075
  y: 0.025
  z: 0.0
D7:
  x: 0.125
  y: 0.025
  z: 0.0
D8:
  x: 0.175
  y: 0.025
  z: 0.0

E1:
  x: -0.175
  y: -0.025
  z: 0.0
E2:
  x: -0.125
  y: -0.025
  z: 0.0
E3:
  x: -0.075
  y: -0.025
  z: 0.0
E4:
  x: -0.025
  y: -0.025
  z: 0.0
E5:
  x: 0.025
  y: -0.025
  z: 0.0
E6:
  x: 0.075
  y: -0.025
  z: 0.0
E7:
  x: 0.125
  y: -0.025
  z: 0.0
E8:
  x: 0.175
  y: -0.025
  z: 0.0

F1:
  x: -0.175
  y: -0.075
  z: 0.0
F2:
  x: -0.125
  y: -0.075
  z: 0.0
F3:
  x: -0.075
  y: -0.075
  z: 0.0
F4:
  x: -0.025
  y: -0.075
  z: 0.0
F5:
  x: 0.025
  y: -0.075
  z: 0.0
F6:
  x: 0.075
  y: -0.075
  z: 0.0
F7:
  x: 0.125
  y: -0.075
  z: 0.0
F8:
  x: 0.175
  y: -0.075
  z: 0.0

G1:
  x: -0.175
  y: -0.125
  z: 0.0
G2:
  x: -0.125
  y: -0.125
  z: 0.0
G3:
  x: -0.075
  y: -0.125
  z: 0.0
G4:
  x: -0.025
  y: -0.125
  z: 0.0
G5:
  x: 0.025
  y: -0.125
  z: 0.0
G6:
  x: 0.075
  y: -0.125
  z: 0.0
G7:
  x: 0.125
  y: -0.125
  z: 0.0
G8:
  x: 0.175
  y: -0.125
  z: 0.0

H1:
  x: -0.175
  y: -0.175
  z: 0.0
H2:
  x: -0.125
  y: -0.175
  z: 0.0
H3:
  x: -0.075
  y: -0.175
  z: 0.0
H4:
  x: -0.025
  y: -0.175
  z: 0.0
H5:
  x: 0.025
  y: -0.175
  z: 0.0
H6:
  x: 0.075
  y: -0.175
  z: 0.0
H7:
  x: 0.125
  y: -0.175
  z: 0.0
H8:
  x: 0.175
  y: -0.175
  z: 0.0

kill1:
  x: 0.255
  y: 0.175
  z: 0.0
        
 
